cartoframes/viz/kuviz.py

```diff
@@ -0,0 +1,47 @@
+from carto.kuvizs import KuvizManager
+from carto.exceptions import CartoException
+
+
+class Kuviz(object):
+    PRIVACY_PUBLIC = 'public'
+    PRIVACY_PASSWORD = 'password'
+
+    def __init__(self, context, id, url, name, privacy=PRIVACY_PUBLIC):
+        self.cc = context
+        self.id = id
+        self.url = url
+        self.name = name
+        self.privacy = privacy
+
+    @classmethod
+    def create(cls, context, html, name, password=None):
+        carto_kuviz = _create_carto_kuviz(context, html, name, password)
+        _validate_carto_kuviz(carto_kuviz)
+        return cls(context, carto_kuviz.id, carto_kuviz.url, carto_kuviz.name, carto_kuviz.privacy)
+
+    @classmethod
+    def all(cls, context):
+        # km = KuvizManager(context.auth_client)
+        # return km.all()
+        pass
+
+    def update(self, html, name, password=None):
+        pass
+
+    def delete(self):
+        pass
+
+
+def _validate_carto_kuviz(carto_kuviz):
+    if not carto_kuviz or not carto_kuviz.url or not carto_kuviz.id or not carto_kuviz.name:
+        raise CartoException('Error creating Kuviz. Something goes wrong')
```

Comment:
Let's be more concrete, otherwise a user won't know what's wrong.

Reply:
We don't really know at this point if it happens. It is only an integration check between different libraries. The same applies to the previous comment

Reply:
Oh OK, I didn't notice this is right after the `Kuviz` has actually been created.

Then this whole thing won't make sense, the KuvizManager should rise an exception (indeed it probably does it already). Remember, we don't want clients of `carto-python` to be checking for None nor invalid results, instead we want to get proper exceptions and catch them.

Reply:
Sorry I answered before reading your comment. Do what you think it's best given how the backend API is designed (but my previous comment would make sense eventually).

Reply:
I think you are right, the check is not needed. Really, this check is here due to https://github.com/CartoDB/carto-python/issues/122 (the KuvizManager parent is not checking is the status code of the response is >=400)

Reply:
For the same reason, I would keep the check until the bug will be solved (when solved the code won't enter on this part of the code

Reply:
OK, let's add a FIXME then, referencing the proper issue, please.

Reply:
Done